kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -47,18 +47,51 @@
 
 _LOGGER: Final = logging.getLogger(__name__)
 
+
+class CustomStep:
```

Comment:
If `__init__` only sets the attributes then this can become a `NamedTuple`:

```suggestion
class CustomStep(NamedTuple):
```